profiles/full.props

```diff
@@ -1,23 +1,23 @@
 <Project>
   <!-- Everything -->
   <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <FakeItEasyTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</FakeItEasyTargetFrameworks>
-    <ValueTaskExtensionsTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</ValueTaskExtensionsTargetFrameworks>
-    <SpecsTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0</SpecsTargetFrameworks>
-    <IntegrationTestsTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0</IntegrationTestsTargetFrameworks>
-    <UnitTestsTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0</UnitTestsTargetFrameworks>
-    <TestHelpersTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</TestHelpersTargetFrameworks>
-    <ApprovalTestsTargetFrameworks>net6.0</ApprovalTestsTargetFrameworks>
-    <RecipesTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0</RecipesTargetFrameworks>
+    <FakeItEasyTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0;net8.0</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</ValueTaskExtensionsTargetFrameworks> <!-- update netX.Y to lowest supported when retiring TFMs -->

```

Comment:
adding net8.0 gave a warning about how it wouldn't be included due to net6.0

Reply:
I'll go back and repro...

Reply:
```
C:\Program Files\dotnet\sdk\8.0.103\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): warning NU5118: File 'C:\Dev\FakeItEasy\src\FakeItEasy.Extensions.ValueTask\obj\Release\net8.0\_._' is not added because the package already cont
ains file 'lib\net6.0\_._' [C:\Dev\FakeItEasy\src\FakeItEasy.Extensions.ValueTask\FakeItEasy.Extensions.ValueTask.csproj]
C:\Program Files\dotnet\sdk\8.0.103\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): warning NU5128: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder do not have exact matches in the o
ther location. Consult the list of actions below: [C:\Dev\FakeItEasy\src\FakeItEasy.Extensions.ValueTask\FakeItEasy.Extensions.ValueTask.csproj]
C:\Program Files\dotnet\sdk\8.0.103\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): warning NU5128: - Add lib or ref assemblies for the net8.0 target framework [C:\Dev\FakeItEasy\src\FakeItEasy.Extensions.ValueTask\FakeItEasy.Ext
ensions.ValueTask.csproj]
```

and technically it's because of the similar lines in `FakeItEasy.Extensions.ValueTask\FakeItEasy.Extensions.ValueTask.csproj`. There's no problem _building_ the project for net8.0 (and I messed up, adding the TFM for Linux). On the other hand, there's no need to build it for net8.0. (Nor net6.0 or even netstandard2.1, I think...)